files/deployable/index.js

```diff
@@ -48,39 +52,46 @@ function getRoleNameFromExecutionARN(arn) {
  */
 async function createAuthenticatorFromConfiguration() {
   const rootLogger = getLogger();
+  var authConfig;
```

Comment:
This shares the same scope but prefer `let` over `var`

```suggestion
  let authConfig;
```